### PR TITLE
V0.11.0 rc

### PR DESCRIPTION
**Pull request by binoy14**
# ~Don't merge until it's tested and we have a changelog following this.~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,44 +25,64 @@
 
 ## Get Started
 
+### Installation
+
 - If you are using
 [create-react-native-app](https://github.com/react-community/create-react-native-app)
-or [Expo](https://expo.io), [follow these instructions](https://github.com/react-native-community/react-native-elements/blob/master/using-with-crna-or-expo.md).
+or [Expo](https://expo.io), [follow these instructions](https://github.com/react-native-training/react-native-elements/blob/v0.11.0/using-with-crna-or-expo.md).
```
Comment (Monte9) [inline README.md]: revert this?

File: README.md [hunk 2]
```diff
@@ -24,44 +25,64 @@
...
 
 - If your project is a standard React Native project (if you have an
-ios/android directory and created it with `react-native init`), [follow these installation instructions](https://github.com/react-native-community/react-native-elements/blob/master/installation.md).
+ios/android directory and created it with `react-native init`), [follow these installation instructions](https://github.com/react-native-training/react-native-elements/blob/v0.11.0/default_installation.md).
```
Comment (Monte9) [inline README.md]: `v0.11.0/default_installation.md` revert this.

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+site_name: React Native Elements UI Toolkit
+site_description: Cross Platform React Native UI Toolkit
+site_url: https://react-native-training.github.io/react-native-elements
+
+# Repository
+repo_name: react-native-training/react-native-elements
+repo_url: https://github.com/react-native-training/react-native-elements
+
+# Copyright
+copyright: Copyright &copy; 2016 Nader Dabit
```
Comment (Monte9) [inline mkdocs.yml]: 2017

File: src/Divider.js [hunk 4]
```diff
@@ -1,18 +0,0 @@
...
-)
-
-styles = StyleSheet.create({
-  container: {
-    height: 1,
-    backgroundColor: colors.grey5
-  }
-})
-
-export default Divider
```
Comment (Monte9) [inline src/Divider.js]: why are we getting rid of this file?

File: src/divider/Divider.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+styles = StyleSheet.create({
+  container: {
+    height: 1,
+    backgroundColor: colors.grey5
+  }
+});
+
+export default Divider;
```
Comment (Monte9) [inline src/divider/Divider.js]: Let's make sure we are exposing this component in `index.js`and also documenting it.

**Comment by Monte9** [comment]
Few comments. Looks good to me overall. 

This PR is a bit hard to review tbh, cause
1. It includes snapshots which we don't really care about
2. It includes the website-docs files which adds a lot of fluff to the PR

Hopefully future release PR's will be easier to review with more component code and less refactoring.

**Comment by bozzmob** [comment]
This surely looks good. But, one comment. Why is there few hundreds of lines of code change which surely doesn't affect the code as such? One such example is adding semi-colons. Was this because we added a new rule to linter or starting using a linter?

**Comment by binoy14** [comment]
@bozzmob Linter was added so the semi colons

**Pull request merged**